Sources/PackageCollections/Providers/JSONPackageCollectionProvider.swift

```diff
@@ -117,9 +117,13 @@ struct JSONPackageCollectionProvider: PackageCollectionProvider {
                                  license: license)
                 }
                 return .init(repository: RepositorySpecifier(url: package.url),
-                             summary: package.description,
+                             description: package.description,
+                             keywords: package.keywords,
                              versions: versions,
-                             readmeURL: package.readmeURL.flatMap { Foundation.URL(string: $0) })
+                             latestVersion: versions.last!,
```

Comment:
I think this should be `.first` since we sort versions in descending order?

Reply:
good catch